### DISCUSSION TITLE
best transformer_flow_type to use?

### DISCUSSION THREAD

**Discussion by bzp83** [p0p4k/vits2_pytorch#93] [Q&A]
Could anyone share which transformer_flow_type yields the best results?

thanks!

**Comment by JohnHerry** [comment]
It is hard to say which is “The Best”. But if you mean to get suggestions about improvements to the VITS or VITS2,  I think you can read the paper about [NaturalSpeech](https://speechresearch.github.io/naturalspeech/) from Microsoft .
